lib/matplotlib/backends/backend_qt5.py

```diff
@@ -613,6 +613,14 @@ def _init_toolbar(self):
             self.layout().setSpacing(12)
             self.setMinimumHeight(48)
```

Comment:
I would get rid of this call, as it is apparently useless, and just add a remark in the comment below.

Othewise, works for me.  (A quick googling didn't yield anything interesting about minimumSize being ignored, although some suggest that QMainWindow may be a bit too eager to manage the size of the QToolbar.)